tests/integration/nm/linux_bridge_test.py

```diff
@@ -141,8 +141,7 @@ def _modify_bridge_options(bridge_config):
     new_conn = nm.connection.ConnectionProfile()
     new_conn.create(iface_bridge_settings)
     conn.update(new_conn)
-    conn.commit(save_to_disk=False, nmdev=nmdev)
-    nm.device.modify(nmdev, conn.profile)
+    nm.device.modify(nmdev, new_conn.profile)
```

Comment:
- The new `update` is now persisting to disk always. This may not be important.
- Seem that now you activate `new_conn` instead of modify `conn`.

Reply:
Unlike only method. The old one is replaced by the new one. The old one became cache only.
This actually takes my several days to debug out.

Reply:
This is intentional. Old profile still hold the old config.